tests/e2e/specs/baseTest.ts

```diff
@@ -60,6 +61,7 @@ export const test = base.extend<TestFixtures>({
 			await fs.promises.cp(logPath, logOutputPath, { recursive: true });
 		}
 	},
+	// eslint-disable-next-line no-empty-pattern
 	createTmpDir: async ({}, use) => {
```

Comment:
Maybe use underscore for unused param?
```suggestion
	// eslint-disable-next-line no-empty-pattern
	createTmpDir: async (_, use) => {
```